pkg/action/action.go

```diff
@@ -225,6 +232,7 @@ func (fa *FrizbeeAction) createPR(ctx context.Context) error {
 			Username: fa.RepoOwner,
 			Password: fa.Token,
 		},
+		Force: true,
```

Comment:
Why the force here? Shouldn't a regular push work if the branch is new?

Reply:
To update the branch if an existing PR was opened. I'm fine removing this and I'm actually glad someone else than me or @rdimitrov joined the reviews, because we were discussing the workflow today and weren't sure what is more accepted by users.

Without force-push I guess the workflow would work like this:
 - a PR is opened by this action
 - someone pushes a new action into a workflow in a separate PR
 - the original PR is merged
 - on next tick of the action (either on schedule or on push to master) a new frizbee PR is created

With this patch:
 - a frizbee PR is created
 - another PR adds a new action
 - the frizbee action is triggered either on schedule or on push to main and force-pushes to the branch, updating the existing PR

WDYT?

Reply:
Ah got it, thanks for the context. I thought you were trying to avoid touching any existing PRs. I got confused because I was expecting to see a rebase, and did not see the commit being recreated elsewhere.

SGTM